src/components/quiz/UserDetailsForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+interface User {
+  name: string | null;
+  displayPic: string | null;
+}
+
+export const UserDetailsForm: React.FunctionComponent = ({status,setStatus}: any) => {
```

Comment:
Please define the prop types! `any` is _very dangerous_ and should _very rarely_ be used!
```suggestion
interface UserDetailsFormProps {
  status: boolean;  // rename this to whatever you rename the other one to
  setStatus: (status: boolean) => void;
}

export const UserDetailsForm: React.FunctionComponent<UserDetailsFormProps> = ({ status, setStatus }) => {
```